tests/Data/CMakeLists.txt

```diff
@@ -52,6 +52,11 @@ target_include_directories(Test_EIP PUBLIC ${CMAKE_SOURCE_DIR}/src)
 target_link_libraries(Test_EIP PUBLIC AccountData Crypto Trie Utils Persistence)
 add_test(NAME Test_EIP COMMAND Test_EIP)
 
+add_executable(Test_DSPowSolution Test_DSPowSolution.cpp)
```

Comment:
Put this section together with Test_POW

Reply:
I believe in context this could make sense, but I think that until now the tests were organized according to the libraries, so you exactly knew where to find the tests. So what I would like to suggest is to even put the _Test_DSPowSolution.cpp_ into tests/Data/MiningData/Test_DSPowSolution.cpp.
Thank you for consideration.